lib/cartopy/mpl/geoaxes.py

```diff
@@ -441,7 +441,7 @@ def hold_limits(self, hold=True):
                     self._autoscaleXon, self._autoscaleYon) = other
 
     @matplotlib.artist.allow_rasterization
-    def draw(self, renderer=None, inframe=False):
+    def draw(self, renderer=None):
```

Comment:
This would break for old versions if someone passed `inframe`, no? We could use `**kwargs` for that.

Reply:
Yes, I was trying to match the call signature of the new `draw()` method, since that is what this is overriding. I added `**kwargs` to the call now and am passing on inframe if it is there for backwards compatibility, but I'm still not sure that is ideal because now we could have people passing in extra keyword arguments as well. I'm open to other thoughts here too.

Note this is probably the most annoying change of the MPL 3.3 update and what made me want to go through this because every pan/zoom will initiate this warning and flood the terminal. So, getting rid of it in some form is desirable.

Reply:
We don't need to explicitly pass `inframe`, just pass `**kwargs` and Matplotlib will take care of the warning/error if the user passed `inframe`, and nothing will happen if they didn't.

Reply:
Ahh, the easy solution ;) don't know why I didn't do that. Thanks!